exchanges/okcoin/okcoin_websocket.go

```diff
@@ -615,7 +643,7 @@ func (o *OKCoin) wsProcessTrades(respRaw []byte) error {
 }
 
 // wsProcessCandles converts candle data and sends it to the data handler
-func (o *OKCoin) wsProcessCandles(respRaw []byte) error {
+func (o *Okcoin) wsProcessCandles(respRaw []byte) error {
```

Comment:
Reasoning for changing the struct name `Okcoin` -> `Okcoin` ?

Reply:
1. It was your suggestion.
2. On their website, they use 'Okcoin' not 'OKCoin'